packages/gui/src/components/FormItemSelect/index.tsx

```diff
@@ -4,13 +4,16 @@ import { FormItem, type FormItemSelectableProps } from "../FormItem";
 export interface FormItemSelectProps<T extends number | string> extends FormItemSelectableProps<T> {
   open?: boolean;
   multiple?: boolean;
+  selectAllText?: string;
+  showCloseButton?: boolean;
```

Comment:
The name of this prop may cause misunderstanding; it's more like an indication of the close-all button in trigger.

And since we already support the `maxDisplayCount` and `maxDisplayText` props that could collapse selections when the selected items are too many, it's fine that we don't support the showCloseButton feature.

Reply:
This is @GuoLei1990 's advice: The close button looks very much like a multiplication symbol, which might confuse users. Hiding the close button makes the interface cleaner, as shown in the image below.
![Uploading image.png…]()

Reply:
After all, The name `showCloseButton` isn't very accurate; `showOptionCloseButton` is more precise.

Reply:
Well, we already support collapsing selections, so the case that he mentioned will not happen again.

![image](https://github.com/user-attachments/assets/c656e147-2651-41e8-af3a-505cf578fbed)

![image](https://github.com/user-attachments/assets/3d986f9d-0bf0-4af2-b28e-cdc9aaa401b5)

And this option is weird: In what case would a user need this? If the number of selections is small, I think the UI is okay. When a developer is forced to show all selections, even when we hide the 'x' symbol, the UI is also disordered and doesn't solve the problem. Plus, I didn't find any component library that supports a multiselect-combobox providing an option like this.

Reply:
And I don't know what a close button symbol should be like that could prevent confusing a user.

Reply:
Agree. Use `maxDisplayCount ` instead.